src/components/buttons/ButtonStyles.tsx

```diff
@@ -74,6 +74,13 @@ export const ButtonStyled = styled.button`
 
 ButtonStyled.displayName = 'ButtonStyled';
 
+export const ButtonIconStyled = styled(ButtonStyled)`
+  padding: 0;
```

Comment:
Should this be set with the `--button-padding` property directly inside BUTTONICON_DIMENSIONS ?